### ISSUE TITLE
Issue in reading tabs in siebel CRM screen using NVDA

### ISSUE THREAD

**Issue by arunksingh16** [nvaccess/nvda#9851]
Hi Team ,

I am facing in issue reading tabs in siebel CRM using NVDA tool. It reads the tab properly outside the citrix environment but it does not read the tab if it is inside citrix env. Why the behavior is different inside or outside the citrix? 
I am stuck with issue and unable to get any support on this. 
Is there a way NVDA support can be purchased or any support can be procured?

Thanks in advance .

**Comment by DrSooom** [comment]
@arun161087: Please completely fill out the bug report draft. Thanks.

**Comment by arunksingh16** [comment]
Hi 
Please find the details.

### Steps to reproduce:
When we open the CRM url hosted in Windows Citrix profile (Virtual windows desktop) issue occurs. Same URL opened in Laptop works fine.

### Actual behavior:
It is not able to read the name of tabs listed when Siebel CRM is opened in browser.

### Expected behavior:
It should be able to read the tabs. 

### System configuration
End users have to work in Citrix desktop. 

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2018.4.1

#### Windows version:
Windows Server 2012

#### Name and version of other software in use when reproducing the issue:
Siebel 8.2 opened in IE11 

#### Other information about your system:

**Comment by JulienCochuyt** [comment]
CC @leonardder

**Comment by Adriani90** [comment]
@arunksingh16 are you still having this issue with NVDA 2020.1?

**Comment by JulienCochuyt** [comment]
@Adriani90 wrote:
> @arunksingh16 are you still having this issue with NVDA 2020.1?

Most likely, yes.
There is no native AT support within Citrix environments.
The reason I CC'ed Leonard back then is because Babbage made it possible to use NVDA in Citrix through a commercial offer, which is by the way mentioned on the [NV Access Partners page](https://www.nvaccess.org/partners/).
For more details, see [UnicornDVC from Access4U](https://access4u.eu/product/unicorndvc/)

**Comment by Adriani90** [comment]
I am working in Citrix environments with NVDA, you need to install NVDA on the virtual machine and to run it from there. Otherwise if you think the performance is too bad while running the virtual machine session only, you need to use Unicorn or another remote add-on, in case your security policies allow for that.
I am closing this issue as works for me. NVDA reads the tabs properly when it runs in the Citrix environment on a virtual machine.
